src/program/wall/spy/spy.lua

```diff
@@ -0,0 +1,147 @@
+module(..., package.seeall)
+
+local lib  = require("core.lib")
+local ipv4 = require("lib.protocol.ipv4")
+local ipv6 = require("lib.protocol.ipv6")
+local util = require("apps.wall.util")
+local scan = require("apps.wall.scanner")
+
+local long_opts = {
+   help    = "h";
```

Comment:
';' instead of ','. I prefer to keep it consistent with the rest of the code base.